### DISCUSSION TITLE
BlueRetro PS2 now only works via usb cable.

### DISCUSSION THREAD

**Discussion by EzDebug** [darthcloud/BlueRetro#670] [Q&A]
Hello everyone I'm starting to say that I'm new to this blueretro. In any case, the product arrives, I put it on my ps2 and it works very well, it allows me to play with the ps4 pad, etc... The next day, however, playing with it for an hour, it keeps disconnecting from the controller, to the point that I think my battery is flat . I take another controller and after a short time the same problem, I reconnect it and suddenly the LEDs on the device go out. I try and try again with the usb cable but nothing doesn't want to turn on anymore. Today looking around I figured out how to update the firmware (since it didn't work with the tool in the guide.) Firmware updated (v1.8.3_hw1) and with the device connected to the PC, it comes back to life. But the problem persists in the ps2, now if I connect it with the usb cable it works, without it it doesn't work anymore. The problem that having the port on the side prevents me from using the second controller. So I was wondering is there a way to make it work directly from the ps2 slot without cable? Since it worked at the beginning?

**Comment by darthcloud** [comment]
Using a regular wired controller does the rumble work?, if not that probably means the PS2 port 8v fuse blown.

**Reply by EzDebug** [reply]
> Using a regular wired controller does the rumble work?, if not that probably means the PS2 port 8v fuse blown.

and I don't think it's the controller slot 1 fuse, since I tried to insert it in 2 too but nothing, in any case how can I test the vibration on the dualshock 2? As I don't have a ps2 pc adapter

**Reply by darthcloud** [reply]
play a fps on your ps2 and shot a gun

**Reply by EzDebug** [reply]
> play a fps on your ps2 and shot a gun

just tried killzone, both shooting and grenades i didn't feel any vibration. Is it difficult to change that fuse?

**Reply by NickGravley** [reply]
Its likely an smt fusr and the difficulty varies by model if you consider some modwls more challenging to gain access to the fuse.  Some models have rf shield soldered on the board, for example (not sure if any cover that fuse), and others just have more case/ribbons with which to contend.  psx-home and sites like that (maybe psx-scene on wayback machine) have maps of the fuse locations if you dont want to just follow your controller port traces.  while you may want rumble in the future, you may temporarily be able to solve the problem on the esp32 side by running vcc power from the 3.3-3.4v instead of rumble power and regulator, and this way its less like to blow the fuse again.

**Reply by EzDebug** [reply]
hi, since i couldn't find the same model of fuse, i opted for 0.7 amp resettable fuses so in case i should be safe. If I can, I'll attach a photo of the sample with the esp32 chip that I have, so maybe you can explain better where to get those 3.3 v from and then bring them to the pin where usually there would be the vibration one.
![20230424_114655](https://user-images.githubusercontent.com/130844399/233962932-187f4fec-6135-4597-bcb3-b46d8d6bb1a9.jpg)
![20230424_114713](https://user-images.githubusercontent.com/130844399/233963001-bc5810e5-f2eb-4024-8f21-f34d4174546e.jpg)

**Reply by ryanschlegl** [reply]
Is that metal prong above the middle pin touching? I'd guess it's a ground and if touching would short circuit.

**Comment by NickGravley** [comment]
the 3.3v on PS2 controller port is the middle pin (whether you look at
connector forward or backward, you can say its pin 5), I've mostly only
used general purpose board,  I am not familiar enough with that unoretro
board to know if the led are run with transistor which may be on 5v
regulation or are directly connected to gpio.  anyway, you may lose more
function going 3.3v but the esp32 and ch340 should be ok.  however someone
more familiar with that board may know what traces you may need to sever if
modding to 3.3v vcc direct, and whether there is a better solder point than
the esp32.  if you have schematic I can maybe help.

**Reply by EzDebug** [reply]
unfortunately I don't have any schematics, as I found it on aliexpress, and obviously it mustn't be done very well if when I inserted it into the console the 2nd time the rumble fuse burned. so now perhaps leaving out this product, do you know if to make one that works 100% I have to do it in diy or is it possible to buy a good one?

**Comment by NickGravley** [comment]
If diy is too intense the modding that board you already have may actually
be less work.  As a lurker here, I see folks have had to do similar mod
with other pre-builts and I've had better success using the usb power or
3.3v in my own diy efforts.  have you considered just making due with a usb
power splitter?

**Comment by NickGravley** [comment]
what model PS2, btw?

**Reply by EzDebug** [reply]
the model is a slim 77004, i'm currently using an original dualshock, since this blue retro has the micro usb port on the right side, it prevents me from inserting a second controller. Since I have small PCBs with a female type c connector, I also thought about inserting perhaps a small type c socket on the back by wiring it directly to the already existing micro usb. so maybe you can use 2 controllers at the same time. to then definitely try once I get those resettable fuses. Ah since you also made the change from 8v to 3.3, by chance did you also have the same problem?

**Reply by NickGravley** [reply]
I used 3.3v for DIY,  but now recall my main reason was just to have compatibility with some PS2->XBOX (original xbox) adapters that did not have the motor voltage, rather than a stability issue.   Theres at least one other discussion on this github about modding one of the other third-party adapters due to a problem with certain version PS1.  Playstation controller extension leads may solve the usb port blocking in a cheap/crude way, while being available for other possible purposes (such as diy).   I don't remember ever buying any that were not fully wired but that's an important detail to check.
